app/src/main/kotlin/compiler/ast/AstNode.kt

```diff
@@ -4,4 +4,78 @@ import compiler.lexer.LocationRange
 
 sealed interface AstNode {
     val location: LocationRange?
+
+    fun printSimple(): String {
+        val stringBuilder = StringBuilder()
+
+        stringBuilder.append(
+            when (this) {
+                is Program.Global -> "<<global definition>>"
+                is Statement -> "<<statement>>"
+
+                is Expression -> when (this) {
+                    is Expression.BinaryOperation -> "${this.leftOperand.printSimple()} ${this.kind} ${this.rightOperand.printSimple()}"
+                    is Expression.BooleanLiteral -> this.value.toString()
+                    is Expression.Conditional -> "${this.condition.printSimple()} ? ${this.resultWhenTrue.printSimple()} : ${this.resultWhenFalse.printSimple()}"
+                    is Expression.FunctionCall -> "${this.name}(${this.arguments.joinToString { it.printSimple() }})"
+                    is Expression.NumberLiteral -> this.value.toString()
+                    is Expression.UnaryOperation -> "${this.kind} ${this.operand}"
+                    is Expression.UnitLiteral -> "nic"
+                    is Expression.Variable -> this.name
+                }
+
+                is Variable -> "${this.kind} ${this.name}: ${this.type}${if (this.value != null) "= ${this.value.printSimple()}" else ""}"
+                is Function -> "czynność ${this.name}(${this.parameters.joinToString { it.printSimple() }}) -> ${this.returnType}"
+                is Function.Parameter -> "${this.name}: ${this.type}${if (this.defaultValue != null) " = ${this.defaultValue.printSimple()}" else ""}"
+                is Expression.FunctionCall.Argument -> "${if (this.name != null) "${this.name} = " else ""}${this.value.printSimple()}"
+                else -> ""
```

Comment:
Do we want this `else` here? It is not needed now (as all possibilities are listed) and if we add new expressions in the future, we probably would like Kotlin compiler to remind us that we need to update this method as well.

Reply:
Oh, I forgot... I had a problem with local compilation, the compiler was complaining that "when is not exhaustive" (very weird, especially since on git, the compilation was fine), and I wanted to compile the project to test it. Then, I forgot to change this back.